File: C/questão 2.c
#include <stdio.h>
#include <stdlib.h>

void main(){
    int nota1, nota2, media, notaex;
    printf("Informe a primeira nota: ");
    scanf("%d", &nota1);

    printf("Digite a segunda nota: ");
    scanf("%d", &nota2);

    media = (nota1 + nota2)/2;

    if(media >= 70){
        printf("Sua média foi: %d. Aprovado!!",media);
    }
    if(media >= 40 && media < 70){
        printf("Sua média foi: %d. Exame final!!\n",media);
        printf("Digite a nota do exame final: ");
        scanf("%d", &notaex);
        if(media + notaex >= 100){
            printf("Aprovado!!");
        }else{
            printf("Reprovado!!");
        }
    }if(media < 40){
        printf("Sua média foi: %d. Reprovado!!",media);
    }
    

}

File: Python/questão 2.py
nota1 = int(input("Informe a 1ª nota: "))
nota2 = int(input("Informe a 2ª: "))

media = (nota1 + nota2) / 2
if media >= 70:
    print("Aprovado")
elif media <= 40:
    print("Terá que fazer o exame final")
    notafinal = int(input("Informe a nota final: "))
    if notafinal >= 70:
        print("Você foi aprovado!!")
    elif notafinal < 70:
        print("Reprovado")
else:
    print("Você está reprovado!!")
